### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by JuannFerrari**
#### Description:
* Adds a basic pull request template for this repository
---

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Title of the PR
+#### Trello/Jira board reference:
+* [Trello/Jira ticket #NUMBEROFTHETICKET](link goes here)
```
Comment (juanmanuelramallo) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
* [Ticket #NUMBER_OF_THE_TICKET](link_goes_here)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Title of the PR
+#### Trello/Jira board reference:
```
Comment (juanmanuelramallo) [inline .github/PULL_REQUEST_TEMPLATE.md:2]:
```suggestion
#### Board:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* [Trello/Jira ticket #NUMBEROFTHETICKET](link goes here)
+---
+#### Description:
+* Add a description of what is the aim of this PR
```
Comment (juanmanuelramallo) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
<!-- * Add a description of what is the aim of this PR -->
```

To make these comments only visible while editing.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+---
+#### Notes:
+*
+---
+#### Tasks:
+- [x] Add each element in this format
+---
+#### Risk:
+Options: (None, little, Medium, Risky)
+* Medium
```
Comment (juanmanuelramallo) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: I'd remove this explanation and just leave for the dev to put in risk whatever they want to.

**Comment by santib** [comment]
I was wondering.. is the intention to have templates whenever a PR is open in this repo? or in the repos that are started from here? (remember this is a template repo)

**Comment by JuannFerrari** [comment]
@santib I think that in both cases. But it's only a suggestion, is not mandatory to follow the PR template

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+## Title of the PR
```
Comment (santib) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Why is the title of the PR needed in the description of the PR ? I mean, it's already above it 🙈

**Pull request merged**